modules/standard/IO.chpl

```diff
@@ -4309,9 +4309,9 @@ config param useNewFileReaderRegionBounds = false;
    A fileReader will never read beyond its maximum end position. In addition,
    reading from a fileReader beyond the end of the underlying file will not
    extend that file.  Reading beyond the end of the file or beyond the end
-   offset of the fileReader will produce the error ``EEOF`` (and return `false`
-   in many cases such as :proc:`fileReader.read`) to indicate that the end was
-   reached.
+   offset of the fileReader will produce the error ``OS.EofError`` (or just
+   return `false` in many cases such as :proc:`fileReader.read`) to indicate
```

Comment:
There is also an overload of `fileReader.read` that returns a `t` and throws on EOF. Did you check that this links to the overload that returns a `bool`?

Reply:
I had not checked as I didn't change the link that was already there -- I checked locally now and it does link to the overload that returns a `bool`.